ports/espressif/boards/waveshare_esp32_s3_zero/pins.c

```diff
@@ -9,6 +9,10 @@
 static const mp_rom_map_elem_t board_module_globals_table[] = {
     CIRCUITPYTHON_BOARD_DICT_STANDARD_ITEMS
 
+    // BOOT button labeled simply as "B" om silkscreen
```

Comment:
```suggestion
    // BOOT button labeled simply as "B" on silkscreen
```

Want to add a B name too?

Reply:
The pins on the board most are labeled simply as 1, 2, 3, etc. and those have aliases of D1, IO1 but not "1", so keeping with the rest of the board, maybe not add the "B". However as I just noted should I instead add IO0 to be consistent with the rest of the aliases?